index.js

```diff
@@ -1,30 +1,44 @@
 const score=document.querySelector(".score")
-let timeLeft=document.querySelector(".time-left")
+const timeLeft=document.querySelector(".time-left")
 const mole=document.querySelectorAll(".mole")
 const box=document.querySelectorAll(".box")
 
 let hitid=null;
+let totalScore=0;
```

Comment:
Score management should be done separately from implementing mole-hitting logic (i.e. hitting-mole is one feature and managing score is another).